enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -288,6 +341,90 @@ func (c *podmanCommandContainer) PullImage(ctx context.Context, creds container.
 	return nil
 }
 
+// cidFilePath returns the path to the container's cidfile. Podman will write
+// the container's ID to this file since we run containers with the --cidfile
+// arg. The file is given the same name as the task workspace directory, plus a
+// ".cid" extension, such as "/tmp/remote_build/{{WORKSPACE_ID}}.cid". Note:
+// this logic depends on the workspace parent directory already existing.
+func (c *podmanCommandContainer) cidFilePath() string {
+	return c.workDir + ".cid"
+}
+
+// monitor starts a goroutine to monitor the container's resource usage. The
+// returned func stops monitoring. It must be called, or else a goroutine leak
+// may occur. Monitoring can safely be stopped more than once. The returned
+// channel should be received from at most once, *after* calling the returned
+// stop function. The received value can be nil if stats were not successfully
+// sampled at least once.
+func (c *podmanCommandContainer) monitor(ctx context.Context) (context.CancelFunc, chan *container.Stats) {
+	ctx, cancel := context.WithCancel(ctx)
+	result := make(chan *container.Stats, 1)
+	go func() {
+		defer close(result)
+		if !c.options.EnableStats {
+			return
+		}
+		var last *container.Stats
+		var lastErr error
+
+		start := time.Now()
+		defer func() {
+			// Only log an error if the task ran long enough that we could reasonably
+			// expect to sample stats at least once while it was executing.
```

Comment:
nit: can you make sure these fall < 80 chars?

Reply:
Done -- I had my editor's tab size at 2. Changed it to 4 to match our repo's `.editorconfig`.